smooks-core/src/main/java/org/milyn/payload/JavaResult.java

```diff
@@ -42,7 +42,7 @@
      * Public default constructor.
      */
     public JavaResult() {
-        this(false);
+        this(true);
```

Comment:
Why this change? Might it cause compatibility issues for people already depending on this behaviour?